src/MBC5.md

```diff
@@ -29,8 +29,10 @@ Same as for MBC1, except that RAM sizes are 8 KiB, 32 KiB and 128 KiB.
 Mostly the same as for MBC1. Writing $0A will enable reading and
 writing to external RAM. Writing $00 will disable it.
 
-Actual MBCs actually enable RAM when writing any value whose bottom 4 bits equal $A (so $0A, $1A, and so on), and disable it when writing anything else.
-Relying on this behavior is not recommended for compatibility reasons.
+Actual MBCs actually enable RAM when writing any value whose bottom 4
+bits equal $A (so $0A, $1A, and so on), and disable it when writing
+anything else. Relying on this behavior is not recommended for
+compatibility reasons.
```

Comment:
```suggestion
Actual MBCs actually enable RAM when writing any value whose bottom 4 bits equal $A (so $0A, $1A, and so on), and disable it when writing anything else.
Relying on this behavior is not recommended for compatibility reasons.
```
Text should not be wrapped at 80 columns (this is a hold-over that we haven't done a pass on); one line per sentence is preferred, as it keeps diffs clearer: a change does not reflow all text following it, muddying `git blame`.

Reply:
> Text should not be wrapped at 80 columns (this is a hold-over that we haven't done a pass on); one line per sentence is preferred, as it keeps diffs clearer: a change does not reflow all text following it, muddying git blame.

Interesting.  We’ve made the opposite decision for MAME’s documentation.  We found that diffs were borderline impossible to deal with when paragraphs weren’t hard wrapped in the source.  I’ll back out this part of the change.

Reply:
If paragraphs aren't wrapped at all, yes, it's a nightmare. This is why I think that "one line per sentence" (which I somewhat borrowed from man pages, where periods at EOL are treated as sentence end markers) is a good compromise.